server/middlewares/error-reporter.js

```diff
@@ -1,15 +1,30 @@
 import debug from 'debug';
-
+import Rollbar from 'rollbar';
 import {
   isHandledError,
   unwrapHandledError
 } from '../utils/create-handled-error.js';
 
+const { ROLLBAR_APP_ID } = process.env;
+
+const rollbar = new Rollbar(ROLLBAR_APP_ID);
 const log = debug('fcc:middlewares:error-reporter');
 
+const errTemplate = ({message, ...restError}, req) => `
+Time: ${new Date(Date.now()).toISOString()}
+Error: ${message}
+Is authenticated user: ${!!req.user}
+Route: ${JSON.stringify(req.route, null, 2)}
+
+${JSON.stringify(restError, null, 2)}
+
+`;
+
 export default function errrorReporter() {
-  if (process.env.NODE_ENV !== 'production' && process.env.ERROR_REPORTER) {
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
I have had this in place to explicitly test the flash messages as and when required. Otherwise the error would get consumed by this.

I understand that its helpful to dump logs when not in production.

Would you prefer  an explicit check `process.env.ERROR_REPORTER !== false` instead? This way you could check the flash messages by disabling the reporter.

Reply:
Oh right. I had no idea what it did and I couldn't find it in the sample.env.

I can place it back in, no problem